server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -642,5 +642,17 @@ export function configureWebSocketServices(
 			}
 			disposers.splice(0, disposers.length);
 		});
+
+		socket.on(
+			"disconnect_document",
+			(clientId: string, documentId: string, errorType?: string) => {
+				if (errorType === undefined) {
+					return;
+				}
+				Lumberjack.error(
+					`Error for client ${clientId}, document ${documentId}: ${errorType}`,
```

Comment:
Instead of burying information in a string message, which requires string parsing in telemetry, it is better to use the 2nd param for Lumberjack logging: properties
```ts
Lumberjack.error("Client disconnected due to error", { clientId, documentId, errorType }, new Error(errorType));
```

The properties dictionary here translates to columns in Kusto for documentId, and a JSON-parseable object for clientId and errorType. Lastly, the Error translates to an `exception` column which will be more easily parsed as well.